backendapi/urls.py

```diff
@@ -6,5 +6,9 @@
 
 urlpatterns = [
     path("nfctag/<hex:tag>", views.scanned),
-    path("test", views.api_get, name="api_get"),
+    path("test/", views.api_get, name="api_get"),
+    path("tickets/verify/<slug:ticket>",
+         views.verify_ticket,
+         name="verify_ticket"),
+    path("attendants/", views.create_attendant, name="create_attendant")
```

Comment:
```suggestion
    path("test", views.api_get, name="api_get"),
    path("tickets/verify/<slug:ticket>",
         views.verify_ticket,
         name="verify_ticket"),
    path("attendants", views.create_attendant, name="create_attendant")
```

no trialing slash on API paths